src/main/java/clickup/ui/pages/SpacePage.java

```diff
@@ -50,7 +58,16 @@ public void addNewSpace(String nameSpace) {
      * @param nameSpace parameter string.
      * @return boolean result.
      */
-    public boolean isFoundNameSpace(String nameSpace) {
+    public boolean isFoundNameSpace(final String nameSpace) {
         return getDriver().getPageSource().contains(nameSpace);
     }
+
+    /**
+     * Method let log out of the page.
+     */
+    public void logOut() {
+        getDriver().findElement(By.xpath("//body")).sendKeys(Keys.ESCAPE);
```

Comment:
Please use Page Factory for this element: "//body"

Reply:
Done.